content/reference/declaration.en.md

```diff
@@ -0,0 +1,538 @@
+---
+title: "Declaration"
+---
+
+# Contributing terms
+
+## Tracking new terms
+
+Tracking terms is done by _declaring_ them and the service they are associated with. Such a declaration is achieved by editing JSON files in the [`declarations`](https://github.com/OpenTermsArchive/contrib-declarations/tree/main/declarations) folder.
+
+Before adding new terms, open the [`declarations`](https://github.com/OpenTermsArchive/contrib-declarations/tree/main/declarations) folder and check if the service you want to track terms for is already declared. If a JSON file with the name of the service is already present, you can jump straight to [declaring terms](#declaring-terms). Otherwise, keep reading!
```

Comment:
This sentence is more appropriate for the Contributing terms page than for the Reference, let's transfer or remove it.

Reply:
I can't seem to find this file named declaration.en.md. Not sure if I removed it or where it is actually situated

Reply:
I can't seem to find this file named declaration.en.md. Not sure if I removed it or where it is actually situated

Reply:
Done. Moved it as part of the introduction of the contributing terms page